contracts/kleros/TalentLayerMultipleArbitrableTransaction.sol

```diff
@@ -129,39 +129,48 @@ contract MultipleArbitrableTransaction is IArbitrable {
         jobRegistryAddress = _jobRegistryAddress;
     }
 
+    function getAmount(uint256 _jobId, uint256 _proposalId) private view returns (uint256){
+        return IJobRegistry(jobRegistryAddress).getProposal(_jobId, _proposalId).rateAmount;
+    }
+
+    function getToken(uint256 _jobId, uint256 _proposalId) private view returns (address){
+        return IJobRegistry(jobRegistryAddress).getProposal(_jobId, _proposalId).rateToken;
+    }
+
+    function getReceiver(uint256 _jobId, uint256 _proposalId) private view returns (address){
+        return IJobRegistry(jobRegistryAddress).getProposal(_jobId, _proposalId).employeeId;
+    }
+
+    function getSender(uint256 _jobId) private view returns (address){
+        return IJobRegistry(jobRegistryAddress).getJob(_jobId).employerId;
+    }
+
     /** @dev Create a ETH-based transaction.
      *  @param _timeoutPayment Time after which a party can automatically execute the arbitrable transaction.
-     *  @param _sender The recipient of the transaction.
-     *  @param _receiver The recipient of the transaction.
      *  @param _metaEvidence Link to the meta-evidence.
      *  @param _adminWallet Admin fee wallet.
      *  @param _adminFeeAmount Admin fee amount.
+     *  @param _jobId the job id in JobRegistry
+     *  @param _proposalId proposal id in JobRegistry
      *  @return transactionID The index of the transaction.
      **/
     function createETHTransaction(
         uint _timeoutPayment,
-        address payable _sender,
-        address payable _receiver,
         string memory _metaEvidence,
-        uint256 _amount,
         address payable _adminWallet,
         uint _adminFeeAmount,
         uint256 _jobId,
         uint256 _proposalId
     ) public payable returns (uint transactionID) {
         require(
-            _amount + _adminFeeAmount == msg.value,
+            getAmount(_jobId, _proposalId) + _adminFeeAmount == msg.value,
             "Fees or amounts don't match with payed amount."
         );
-        //address(this).transfer(msg.value); Need to look up
+        address(this).transfer(msg.value);
```

Comment:
you don't need it, as it's payable, funds are already transfered to the contract